apollo-federation/src/operation/directive_list.rs

```diff
@@ -332,6 +335,18 @@ impl DirectiveList {
         inner.rehash();
         Some(item)
     }
+
+    /// Removes @defer directive from self if it has a matching label.
+    pub(crate) fn remove_defer(&mut self, defer_labels: &IndexSet<String>) {
```

Comment:
This was lifted out of the old `DeferFilter`. We don't need to pass schema anymore as at this stage all `@defer` applications will already have labels normalized.